examples/create_postcards_from_csv/postcards.py

```diff
@@ -0,0 +1,152 @@
+# usage:
+#   python postcard.py input.csv
+#   logs to csv's in the output directory
+
+import csv
+import datetime
+import lob
+import os
+import sys
+
+sys.path.insert(0, os.path.abspath(__file__+'../../..'))
+
+###########################################################
+# TODO: Provide your API Key, keep this secure
+lob.api_key = 'test_fc26575412e92e22a926bc96c857f375f8b'  # '<your key here>'
+
+# TODO: Create your from_address
+try:
+    from_address = lob.Address.create(
+            name='Your Name',
+            company='Your Company',
+            address_line1='123 Test Avenue',
+            address_line2='STE 456',
+            address_city='San Francisco',
+            address_state='CA',
+            address_zip='94107'
+            )
+except Exception, e:
+    print('Error: ' + str(e))
+    print('Failed to create from_address.')
+    sys.exit(1)
+
+###########################################################
+
+# Input check
+if len(sys.argv) < 2:
+    print("Please provide an input CSV file as an argument.")
+    print("usage: python letter.py <CSV_FILE>")
+    sys.exit(1)
+
+input_filename = sys.argv[1]
+
+success_csv_fields = [
+    'name',
+    'id',
+    'url',
+    'address_line1',
+    'address_line2',
+    'address_city',
+    'address_state',
+    'address_zip'
+]
+errors_csv_fields = ['error']
+
+# create the output directory,
+output_dir = os.path.join('.',  'output')
+if not os.path.isdir(output_dir):
+    os.mkdir(output_dir)
+
+timestamp = datetime.datetime.now().strftime('%Y-%m-%d_%H-%M-%S')
+
+try:
+    output_dir = os.path.join(output_dir, timestamp)
+    os.mkdir(output_dir)
+except Exception, e:
+    print('Error: ' + str(e))
+    print('Failed to create output directory. Aborting all sends.')
+    sys.exit(1)
+
+# output csv names
+success_filename = os.path.join(output_dir, 'success.csv')
+errors_filename = os.path.join(output_dir, 'errors.csv')
+
+with open('postcard_front.html', 'r') as front_file,\
```

Comment:
It became a really long line, but I couldn't think of anything better besides nesting the with statements. Which is still possible!